discord_bots/commands.py

```diff
@@ -2220,131 +2222,43 @@ async def trueskill(ctx: Context):
-        )
-        for ipg_channel in ipg_channels or []:
-            discord_channel: discord.abc.GuildChannel | None = guild.get_channel(
-                ipg_channel.channel_id
-            )
-            if isinstance(discord_channel, discord.VoiceChannel):
-                # This is suboptimal solution but it's good enough for now. We should keep track of each team's VC in the database
-                if discord_channel.name == in_progress_game.team0_name:
-                    be_voice_channel = discord_channel
-                elif discord_channel.name == in_progress_game.team1_name:
-                    ds_voice_channel = discord_channel
-
-        coroutines = []
-        for player in team0_players:
-            if player.move_enabled and be_voice_channel:
-                member: Member | None = guild.get_member(player.id)
-                if member:
-                    try:
-                        coroutines.append(
-                            member.move_to(
-                                be_voice_channel,
-                                reason=f"Game {game_id} started",
-                            )
-                        )
-                    except Exception:
-                        _log.exception(
-                            f"Caught exception moving player to voice channel"
-                        )
-
-        for player in team1_players:
-            if player.move_enabled and ds_voice_channel:
-                member: Member | None = guild.get_member(player.id)
-                if member:
-                    try:
-                        coroutines.append(
-                            member.move_to(
-                                ds_voice_channel,
-                                reason=f"Game {game_id} started",
-                            )
-                        )
-                    except Exception:
-                        _log.exception(
-                            f"Caught exception moving player to voice channel"
-                        )
-    # use gather to run the moves concurrently in the event loop
-    # note: a member has to be in a voice channel already for them to be moved, else it throws an exception
-    results = await asyncio.gather(*coroutines, return_exceptions=True)
-    for result in results:
-        # results should be empty unless an exception occured when moving a player
-        if isinstance(result, BaseException):
-            _log.exception("Ignored exception when moving a gameplayer:")
-
-
-@bot.command(usage="<game_id>")
+@bot.tree.command(
+    name="movegameplayers",
+    description="Moves players in an in progress game to their respective voice channels",
+)
 @commands.check(is_admin)
-async def movegameplayers(ctx: Context, game_id: str):
+async def movegameplayers(interaction: Interaction, game_id: str):
     """
     Move players in a given in-progress game to the correct voice channels
     """
-    message = ctx.message
+    assert interaction.guild
```

Comment:
Along with this assertion for type checking, add the `discord.add_commands.guild_only()` decorator to ensure that this command can only be run inside a guild